tests/installation/select_patterns_and_packages.pm

```diff
@@ -157,6 +157,23 @@ sub run {
     my ($self) = @_;
     my $dep_issue;
     $self->gotopatterns;
+    # select all patterns via menu and end in graphical mode
+    if (check_var('PATTERNS', 'all') && !check_var('VIDEOMODE', 'text')) {
+        # move mouse on patterns and open menu
+        mouse_set 100, 400;
```

Comment:
different screen resolutions which we might have for different arches ? maybe we could use here assert_and_click ?

Reply:
true, but since when we use another resolution than 1024x768 ?
I think we don't, but would be nice needling time :smiling_imp:

Reply:
I am not sure but I thought that it differs by architecture. e.g. some arm workers can have a different one?

Reply:
no, do you have idea how much needles it would be ?